model.py

```diff
@@ -192,7 +192,7 @@ def fit(self, pixel, spec_obs, par, sig=[], **kwargs):
 
         pnew = par + dict(zip(varykeys, params))
         # attach uncertainties
-        for k,v in zip(varykeys, np.diag(e_params)):
+        for k,v in zip(varykeys, np.sqrt(np.diag(e_params))):
```

Comment:
This now converts directly to uncertainty. It was an unnoticed bugfix (only e_rv was intented to be set for the legend).

The old version has written the variance (while header e_param implied uncertainties)!
https://github.com/mzechmeister/viper/blob/fc11305dfb817aa7c90fcc7cab53ec3ed2315fdd/viper.py#L833